src/components/screens/monitor/delegates/forgingDetails.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Box from '../../../toolbox/box';
+import BoxHeader from '../../../toolbox/box/header';
+import BoxContent from '../../../toolbox/box/content';
+import styles from './forgingDetails.css';
+import LiskAmount from '../../../shared/liskAmount';
+import { tokenMap, initialLSKSupply } from '../../../../constants/tokens';
+
+const ForgingDetails = ({
+  t, delegates, sortDirection, networkStatus,
+}) => {
+  // When sorting changes, delegates.data is reversed
+  // TODO: Check for alternative solution
+  const delegatesList = sortDirection.includes('asc') ? [...delegates.data] : [...delegates.data].reverse();
+  const totalForged = networkStatus.data.supply - initialLSKSupply;
+
+  const lastForger = [...delegatesList].sort(
+    (a, b) =>
+      b.lastBlock && a.lastBlock && b.lastBlock.height - a.lastBlock.height,
+  )[0];
+
+  const nextForgers = [...delegatesList]
```

Comment:
💡 I think this would solve the glitch when loading the page:
```suggestion
  const nextForgers = [...delegatesList].filter(d => d.forgingTime)
```
The problem is that `forgingTime` comes from different api request thatn the list of delegates, so at some point it's undefined

Reply:
Thank you, Vit! I used your suggestion and it works better. There is a small delay but I think it's inevitable...